### PR TITLE
DMEERX-406 cql practitioner

### PR DESCRIPTION
**Pull request by yunwwang**
Create CQL module BasicPractitionerInfo
Update SQL rules to use this module.

### CODE REVIEW

File: server/src/main/jib/smartAppFhirArtifacts/BasicPractitionerInfoPrepopulate-R4.cql [hunk 1]
```diff
@@ -0,0 +1,36 @@
+library BasicPractitionerInfo version '0.0.1'
```
Comment (hossenlopp) [inline server/src/main/jib/smartAppFhirArtifacts/BasicPractitionerInfoPrepopulate-R4.cql]: This should have a different version number from the STU3 version of this library.

File: server/src/main/jib/smartAppFhirArtifacts/library-basic-practitioner-info-prepopulate-r4.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+  "resourceType": "Library",
+  "id": "urn:hl7:davinci:crd:library-basic-practitioner-info-r4",
+  "version": "0.0.1",
```
Comment (hossenlopp) [inline server/src/main/jib/smartAppFhirArtifacts/library-basic-practitioner-info-prepopulate-r4.json]: Like the CQL version comment. This should be different than the R3 version.

File: server/src/main/jib/smartAppFhirArtifacts/BasicPractitionerInfoPrepopulate-R4.cql (hunk 1 above)
Comment (yunwwang) [reply server/src/main/jib/smartAppFhirArtifacts/BasicPractitionerInfoPrepopulate-R4.cql]: fixed

File: server/src/main/jib/smartAppFhirArtifacts/library-basic-practitioner-info-prepopulate-r4.json (hunk 2 above)
Comment (yunwwang) [reply server/src/main/jib/smartAppFhirArtifacts/library-basic-practitioner-info-prepopulate-r4.json]: fixed

**Pull request merged**